core/domain/param_domain_test.py

```diff
@@ -21,9 +21,31 @@
 import utils
 
 
-class ParameterDomainUnitTests(test_utils.GenericTestBase):
+class ParamSpecUnitTests(test_utils.GenericTestBase):
     """Tests for parameter domain objects."""
+    # Unittest to test to_dict function.
+    def test_to_dict(self):
+        initial_vals = {
+            'obj_type': 'UnicodeString',
+        }
+        param_spec_obj = param_domain.ParamSpec('UnicodeString')
+        # ParamSpec.from_dict(initial_vals)
```

Comment:
Why this comment?

\cc @oppia/dev-workflow-team

Reply:
Removed

Reply:
@apb7 Could this be made into a lint check? I am not sure about this. PTAL.
Thanks.